### PR TITLE
chore: sync ap migrations

### PR DESCRIPTION
**Pull request by kamilczaja**

### CODE REVIEW

File: extensions/catalog-crawler/catalog-crawler-db/src/main/resources/db-crawler/migration/V10__White_Label_Refactor.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+alter table organization
+    rename column mds_id to id;
+
+alter table "user"
+    rename column organization_mds_id to organization_id;
```
Comment (ununhexium) [inline extensions/catalog-crawler/catalog-crawler-db/src/main/resources/db-crawler/migration/V10__White_Label_Refactor.sql:5]: Changed but not updated in the code? Why? If it's not used, why does it exist?

File: extensions/catalog-crawler/catalog-crawler-db/src/main/resources/db-crawler/migration/V10__White_Label_Refactor.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+alter table connector
+    rename column mds_id to organization_id;
+
+alter table connector
```
Comment (ununhexium) [inline extensions/catalog-crawler/catalog-crawler-db/src/main/resources/db-crawler/migration/V10__White_Label_Refactor.sql:10]: Changed but not updated in the code? Why? If it's not used, why does it exist?

File: extensions/catalog-crawler/catalog-crawler-db/src/main/resources/db-crawler/migration/V10__White_Label_Refactor.sql [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    rename column provider_mds_id to provider_organization_id;
+
+alter table component
```
Comment (ununhexium) [inline extensions/catalog-crawler/catalog-crawler-db/src/main/resources/db-crawler/migration/V10__White_Label_Refactor.sql:13]: Changed but not updated in the code? Why? If it's not used, why does it exist?

**Review by ununhexium — CHANGES_REQUESTED** [review]
Something seems missing or extra with this columns renaming.
Is it used externally? Not in this code base?

**Comment by kamilczaja** [comment]
Copied to have this documented:

so to clear out your doubts
 
most of the stuff in the crawler migration is not used in the crawler
 
but it still needs to be there because the database schema needs to look exactly as in the authority portal. remember, the crawler talks directly to the AP database

**Review by ununhexium — APPROVED** [review]
LGTM.

**Pull request merged**